src/Examine.Core/Search/IFacetAppending.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Examine.Search
+{
+    /// <summary>
+    /// Allows for appending more operations
+    /// </summary>
+    public interface IFacetAppending
+    {
+        /// <summary>
+        /// Allows for adding more operations
+        /// </summary>
+        /// <returns></returns>
+        IOrdering And();
```

Comment:
It would be fine to encapsulate the facting in a callback. But if we remove the `And()` then we lose the ability to chain `WithFacet()` statements because different `WithFacet()` allow different modifiers like `MaxCount()` and `FacetField()`.
Then your example would be forced to be:

```csharp
var query = searcher.CreateQuery("content")
             .Field("nodeName", "sydney")
             .OrderBy(new SortableField("sortOrder", SortType.Int))
             .WithFacets(facets => 
                   {
                       facets.WithFacet("nodeName");
                       facets.WithFacet("test");
                   }
              )
             .Execute();
```

Reply:
I've refactored to encapsulate the facets now and can see that the double `WithFacet` naming now is a bit much. Maybe the `WithFacet` methods should be named just `Facet` again? Then the example would be:

```csharp
var query = searcher.CreateQuery("content")
             .Field("nodeName", "sydney")
             .OrderBy(new SortableField("sortOrder", SortType.Int))
             .WithFacets(facets => 
                   {
                       facets.Facet("nodeName");
                       facets.Facet("test");
                   }
              )
             .Execute();
```

Reply:
Yep the double WithFacet is too many Withs :) for the chaining, can't the Facet method just return IFaceting so you can do:

```cs
facets.Facet("nodeName").Facet("test")
```